www/toptable.mako

```diff
@@ -0,0 +1,169 @@
+                    Public Library of Science (PLOS)<br>
+                    Riffyn<br>
+                    rOpenSci<br>
+                    Software Sustainability Institute<br>
+                    Society for a Science of Clinical Psychology<br>
+                    Society of Australasian Social Psychologists<br>
+                    SPARC<br>
+                    Sunlight Foundation<br>
+                    Time-series Experiments in Social Sciences<br>
+                    Ubiquity Press<br></p>
+
+                <hr>
+                <h1><strong>Related Efforts</strong></h1>
+                    <p class="margin-top-20">The guidelines are the output of a meeting held in November 2014, organized by the <a href="http://bitss.org/">Berkeley Initiative for Transparency in the Social Sciences</a>, <a href="http://www.sciencemag.org"><em>Science</em> Magazine</a>, and the <a href="http://cos.io">Center for Open Science</a>.  TOP Guidelines are one of a family of efforts aiming to increase openness, integrity, and reproducibility of scientific research.  Explore and join these related efforts:</p>
+                        <ul>
+                            <li><a href="http://www.apa.org/science/leadership/bsa/data-sharing-report.pdf">APA Board of Scientific Affairs Statement on Data-Sharing (June, 2015)</a></li>
+                            <li><a href="https://osf.io/tvyxz/wiki/home/">Badges to acknowledge open practices</a></li>
+                            <li><a href="http://www.copdess.org/">COPDESS: Coalition for Publishing Data in the Earth and Space Sciences</a></li>
+                            <li><a href="http://www.dartstatement.org/">DA-RT: Data Access and Research Transparency</a></li>
+                            <li><a href="http://www.nih.gov/about/reporting-preclinical-research.htm">NIH Principles and Guidelines for Reporting Preclinical Research</a></li>
+                            <li><a href="https://osf.io">The Open Science Framework</a></li>
+                            <li><a href="http://centerforopenscience.org/prereg/">The Pre-reg Prize</a></li>
+                            <li><a href="https://osf.io/8mpji/wiki/home/">Registered Reports</a></li>
+                        </ul>
+                  
+                
+                <!-- END INFO BLOCK -->
+            </div>
+        </div>
+        <!--End Container-->
+    </div>
+    <!-- END PAGE CONTAINER -->
+
+    <!-- BEGIN PAGE LEVEL JAVASCRIPTS(REQUIRED ONLY FOR CURRENT PAGE) -->
+    <%def name="javascript_bottom()">
+
+            <script type="text/javascript" src="../static/plugins/fancybox/source/jquery.fancybox.pack.js"></script>
+            <script src="../static/plugins/jquery.mixitup.min.js"></script>
+            <script src="../static/scripts/app.js"></script>
+            <script src="../static/scripts/portfolio.js"></script>
+                        <script type="text/javascript">
+                jQuery(document).ready(function() {
```

Comment:
same with this JQuery stuff - this was copied from another mako file and does nothing here